CHANGELOG.md

```diff
@@ -1,6 +1,11 @@
 # Changelog
 
 ## Unreleased
+- Updated the README and CONTRIBUTING docs
+  ([#638](https://github.com/census-instrumentation/opencensus-python/pull/638))
```

Comment:
This one probably doesn't need a mention in the changelog.

Reply:
Got it, removed.